changelog.html

```diff
@@ -56,6 +56,9 @@
 <div id="trunk" style="display:none"><!--=TRUNK-BEGIN=-->
 <ul class=image>
   <li class=>
```

Comment:
removing it makes merging harder if there are competing pull requests that land first